src/Iot.Device.Bindings/Utils/BitmapImage.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
```

Comment:
1. copyright header is missing
2. this should be internal in your device, for generic purpose APIs please create an issue and after we get an agreement from other people on APIs then we can move it to shared directory or common folder in the devices

Reply:
I moved it to Common as requested by @joperezr. You can create an issue. But this is actually part of my other work item, porting Adafruit GFX lib. So maybe we could use that issue instead. For now I drop it in Common, once GFX lib is merged it will have found its final resting place.